docs/routing.md

```diff
@@ -144,7 +144,36 @@ route.json
       "host": "hello-openshift.v3.rhcloud.com",
       "serviceName": "hello-openshift"
     }
-    
+
+## Securing Your Routes
+
+Creating a secure route to your pods can be accomplished by specifying the TLS Termination of the route and, optionally,
+providing certificates to use.  TLS Termination falls in the following configuration buckets:
+
+#### Edge Termination
+Edge termination means that TLS Termination occurs prior to traffic reaching the pod.  TLS certificates are served by either 
+inspecting the SNI host sent in the request and finding a certificate for that host (matching the `CommonName` in the 
+certificate file) or, for non-SNI requests by serving a generic certificate.
+
+Edge termination is configured by setting `TLS.Termination` to `edge` on your `route` and by specifying the `CertificateFile`
+and `KeyFile` (at a minimum).  
+
+#### Pod Termination
+Pod termination is a passthrough mechanism to send encrypted traffic straight to your pod.  Pod termination relies on 
+SNI hosts to determine where to route the traffic.  
+
+Pod termination is configured by setting `TLS.Termination` to `pod` on your `route`.  No other information is required.
```

Comment:
I don't think we want to couple routes to pods - instead this should just refer to "backend" or whatever the term you're using is.